wasm/wasm_source/src/tx_init_proposal.rs

```diff
@@ -41,12 +41,13 @@ fn apply_tx(ctx: &mut Ctx, tx: Tx) -> TxResult {
                 .map_err(|err| {
                     ctx.set_commitment_sentinel();
```

Comment:
Maybe in a future PR we could also get rid of this other sentinel just like you did for the vps

Reply:
https://github.com/anoma/namada/issues/2962